### DISCUSSION TITLE
Please help me install  DisplayCal - Py3  on Kubuntu 22.04

### DISCUSSION THREAD

**Discussion by Danas-Anis** [eoyilmaz/displaycal-py3#196] [Q&A] [answered]
Hello, 

I am on Kubuntu 22.04 OS and am trying to install DisplayCal Py3, but I always fail. 
I fallow the given instruction on Github, unfortunately I am not skilled at programing and would really appreciate dummy proof assistance to get it running. 
As I understand the installation packages are under way, but it is not clear when they will be available, maybe there is a ppa that would allow easier way of installing? 

Here is an error that I get when trying to install: 

` pip install DisplayCAL
Defaulting to user installation because normal site-packages is not writeable
Collecting DisplayCAL
  Using cached DisplayCAL-3.9.8.tar.gz (10.7 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
  Preparing metadata (pyproject.toml) ... done
Collecting numpy
  Using cached numpy-1.23.4-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (17.1 MB)
Requirement already satisfied: dbus-python in /usr/lib/python3/dist-packages (from DisplayCAL) (1.2.18)
Collecting wxPython
  Using cached wxPython-4.2.0.tar.gz (71.0 MB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [8 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-efp851ui/wxpython_ee6a816496de41c99172b3a1cd7de4c5/setup.py", line 27, in <module>
          from buildtools.config import Config, msg, opj, runcmd, canGetSOName, getSOName
        File "/tmp/pip-install-efp851ui/wxpython_ee6a816496de41c99172b3a1cd7de4c5/buildtools/config.py", line 30, in <module>
          from attrdict import AttrDict
      ModuleNotFoundError: No module named 'attrdict'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.`
I find two new folders created: displaycal-py3 and displaycal_venv in my Home directory, however it appears to be unhappy about something still missing etc. 

Thank you very much for your help! 
Kind regards.

**Comment by p5k369** [comment, marked as answer]
Hey @Danas-Anis I think there will be no official package for ubuntu until the naming issue isnt sorted out. You can read about it here: #96.
The failure you get can be solved by installing `attrdict3` prior to displaycal. Your problem is the same as described in #155. In the next release this won't be an issue anymore.

**Reply by Danas-Anis** [reply]
Hello! 
My biggest gratitude! I was able to install DisplayCal! 
It might be useful to mention this fix in the main page, as I understand this is occurring issue, mentioning this issue and solution could lead to less confused poor souls lurking around seeking for help. How come I didn't find that discussion?  I need to up and improve my searching skills. 

Thank you very much! You saved my day!

**Reply by p5k369** [reply]
Hey @Danas-Anis, good to hear. Unfortunately this isnt a real fix, it's more of a workaround that has nothing to do with displaycal. As I mentioned this workaround is already part of the dev branch and will be part of the next release. In the end that has to be fixed in the wxpython package, see https://github.com/wxWidgets/Phoenix/issues/2225
